package.json

```diff
@@ -47,6 +47,9 @@
   },
   "dependencies": {
     "@emotion/react": "11.10.6",
+    "apg-js": "^4.1.3",
```

Comment:
Looks like the other dependencies are version locked for this repo - can we do the same for these additional dependencies?

Reply:
good catch!

Reply:
Good catch. Done!